Analysis/ScanLibraries/include/XiaData.hpp

```diff
@@ -89,7 +89,9 @@ class XiaData {
     ///@return True if this channel was generated on the module
     bool IsVirtualChannel() const { return isVirtualChannel_; }
 
-    ///@return The baseline as it was calculated on the module
+    ///@return The baseline as it was calculated on the module.
```

Comment:
The notes about what the function returns shoudl be above this line. The units here are in trapezoidal filter units, whatever they are. Make sure that you note that the baseline that's provided here cannot be used in conjunction with the QDC or subtracted from the trace as we do with the baseline calculated from the trace.